src/python/pants/jvm/util_rules.py

```diff
@@ -29,6 +28,12 @@ async def digest_to_file_digest(request: ExtractFileDigest) -> FileDigest:
         )
 
     file_info = digest_entries[0]
+
+    if not isinstance(file_info, FileEntry):
+        raise AssertionError(
+            f":Unexpected error: '{request.file_path}' refers to a directory, " "not a file."
```

Comment:
```suggestion
            f"Unexpected error: '{request.file_path}' refers to a directory, not a file."
```

Reply:
I think the right bit of that was me pre-empting Black, but apparently the error message wasn't long enough. Fixing.